nicenumber/nicenumber.py

```diff
@@ -123,8 +127,37 @@ def to_numeric(string:str, family:str = 'number'):
     >>> to_numeric(1.5M)
     1500000
     """
+    def raise_err(err):
```

Comment:
move this function outside as a common function

Reply:
Moved in this pr